bindings/java/src/main/java/org/openucx/jucx/ucp/UcpEndpoint.java

```diff
@@ -235,6 +249,12 @@ public UcpRequest recvStreamNonBlocking(long localAddress, long size, long flags
         return recvStreamNonBlockingNative(getNativeId(), localAddress, size, flags, callback);
     }
 
+    public UcpRequest recvStreamNonBlocking(long[] localAddresses, long[] sizes, long flags,
+                                            UcxCallback callback) {
+        return recvStreamIovNonBlockingNative(getNativeId(),
+            localAddresses, sizes, flags, callback);
```

Comment:
wrap line

Reply:
We use 4 spaces in java: https://github.com/openucx/ucx/blob/master/bindings/java/checkstyle.xml#L62 for long methods calls.

Reply:
yes, but seems "localAddress" could be on prev line